brambling/utils/model_tables.py

```diff
@@ -506,6 +507,28 @@ def confirmed(self, obj):
     environment_avoid = related_objects_list("ef_avoid")
     environment_cause = related_objects_list("ef_cause")
```

Comment:
You should be able to remove these related_objects_list lines now that it's not being used below.